setup/install_upstream_rust_keylime/test.sh

```diff
@@ -67,38 +55,34 @@ _EOF'
         rlRun "cp tests/actions/shim.py /usr/libexec/keylime"
```

Comment:
Not sure if the shim related part is still necessary. The upstream issue is still open.

Reply:
Do you have link to this upstream issue?

Reply:
https://github.com/keylime/rust-keylime/issues/325 It is mentioned in a comment above.